Builds for OpenShift Release Notes 1.1.1 (#87311)
openshift/openshift-docs: modules/ob-release-notes-1-1-1.adoc

// This module is included in the following assemblies:
// * about/ob-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="ob-release-notes-1-1-1_{context}"]
= Release notes for {builds-shortname} General Availability 1.1.1

> {builds-shortname} General Availability (GA) 1.1.1 is now available on {ocp-product-title} 4.16.

Author: Thank you, I really like the suggestion, I will incorporate it to other places as well :+1:

[id="fixed-issues-1-1-1_{context}"]
== Fixed issues

The following section highlights fixed issues in {builds-shortname} 1.1.1.

> * With this update, the Shared Resource Container Storage Interface (CSI) Driver has permissions to create subject access reviews and get/list/watch Shared Resource objects.

Reviewer: ```suggestion
* With this update, the Shared Resource Container Storage Interface (CSI) Driver has the following permissions:
* Create subject access reviews
* Get, list, or watch Shared Resource objects
```

This is an optional suggestion depending upon whether `Get, list, and watch` mean the same thing.

// This module is included in the following assemblies:
// * about/ob-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="ob-release-notes-1-1-1_{context}"]
= Release notes for {builds-shortname} General Availability 1.1.1

> With this update, {builds-shortname} 1.1.1 is now Generally Available (GA) on {ocp-product-title} 4.12, 4.13, 4.14, 4.15, and 4.16.

Reviewer 2: See https://github.com/openshift/openshift-docs/pull/87313#discussion_r1933754632

Reviewer 2: Applies to all instances
